### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by cjdduarte** [lovac42/HoochieBaby#2]
follows the error. It does not always occur (50%)

```
Anki 2.1.7 (a6c34fd7) Python 3.6.7 Qt 5.12.0 PyQt 5.11.3
Platform: Windows 10
Flags: frz=True ao=True sv=2

Caught exception:
  File "aqt\webview.py", line 27, in cmd
  File "aqt\webview.py", line 85, in _onCmd
  File "aqt\webview.py", line 346, in _onBridgeCmd
  File "aqt\overview.py", line 37, in _linkHandler
  File "aqt\main.py", line 469, in moveToState
  File "aqt\main.py", line 499, in _reviewState
  File "aqt\reviewer.py", line 45, in show
  File "aqt\reviewer.py", line 86, in nextCard
  File "anki\schedv2.py", line 45, in getCard
  File "<decorator-gen-68>", line 2, in _getCard
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 71, in repl
  File "D:\Anki\ankidata21\addons21\1847358755\hoochieBaby.py", line 41, in getCard
    c = self._getLrnDayCard()
  File "anki\schedv2.py", line 528, in _getLrnDayCard
  File "anki\collection.py", line 258, in getCard
  File "anki\cards.py", line 31, in __init__
  File "anki\cards.py", line 68, in load
  File "anki\db.py", line 74, in first
  File "anki\db.py", line 32, in execute
<class 'sqlite3.InterfaceError'>: Error binding parameter 0 - probably unsupported type.
```

**Comment by lovac42** [comment]
yeah that's already fixed, please update to the latest version 0.0.4. thanks.

**Comment by cjdduarte** [comment]
Tanks. Work.
